<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>903</width>
    <height>695</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: 'green';</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="dealerCard1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>60</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard1">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>350</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="shuffleButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>590</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'light gray'</string>
    </property>
    <property name="text">
     <string>Shuffle start / Over</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hitMeButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>590</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'light gray'</string>
    </property>
    <property name="text">
     <string>Hit Me!</string>
    </property>
   </widget>
   <widget class="QLabel" name="dealerTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dealer</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="playerTitle">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Player</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="standButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>590</y>
      <width>150</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: 'light gray'</string>
    </property>
    <property name="text">
     <string>Stand</string>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>60</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>60</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="dealerCard5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>60</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_clubs.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>350</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard4">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>350</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="playerCard5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>350</y>
      <width>150</width>
      <height>218</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>images/cards/2_of_diamonds.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>903</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
